src/Microsoft.Framework.Runtime/NuGet/Repositories/PackageRepository.cs

```diff
@@ -108,6 +108,14 @@ public IEnumerable<PackageInfo> FindPackagesById(string packageId)
                         continue;
                     }
 
+                    var manifestFilePath = _repositoryRoot.GetFiles(versionDir, "*" + Constants.ManifestExtension)
```

Comment:
After introducing lock file, this logic is never executed during runtime of your apps. It only happens during `dnu restore`.